examples/gke-public-cluster/main.tf

```
@@ -52,6 +52,10 @@ module "gke_cluster" {
   alternative_default_service_account = var.override_default_node_pool_service_account ? module.gke_service_account.email : null
 
   enable_vertical_pod_autoscaling = var.enable_vertical_pod_autoscaling
+
+  resource_labels = {
+    purpose = "testing"
```

```suggestion
    environment = "testing"
```
Let's change `purpose` to `environment` as it's likely a better example for most users.

---

I have pushed your change here: https://github.com/gruntwork-io/terraform-google-gke/pull/98/commits/4f14a759b0568252671448706c0668e0eff53d3c